### PR TITLE
Added: support `nodejs` 7.

### PR DESCRIPTION
**Pull request by evilebottnawi**
> Which issue, if any, is this issue related to?

Support new version of `nodejs`.

> Is there anything in the PR that needs further explanation?

No, it's self explanatory.

### CODE REVIEW

**Comment by ntwb** [comment]
For reference: https://github.com/nodejs/lts

NodeJS 4.x and 6.x branches are both "LTS" (Long Time Support)

NodeJS 7.x is the "stable" branch

**Comment by ntwb** [comment]
I'm hesitant to suggest this... Should we also add NodeJS 7 for Appveyor?

**Comment by evilebottnawi** [comment]
@ntwb yep i think we should, 7 version have breaking changes and in theory can break

**Comment by evilebottnawi** [comment]
@davidtheclark what do you think about `appveyor`?

**Comment by davidtheclark** [comment]
Go for it

File: appveyor.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 environment:
   matrix:
-    - nodejs_version: 6
+    - nodejs_version: 7
```
Comment (ntwb) [inline appveyor.yml]: Let's test both the LTS 6.x branch and 7.x

Comment (evilebottnawi) [inline appveyor.yml]: @ntwb done

**Pull request merged**